src/OpenLoco/GameState.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+#include "Company.h"
+#include "Entities/Entity.h"
+#include "Industry.h"
+#include "Map/Animation.h"
+#include "Map/Wave.h"
+#include "Station.h"
+#include "Town.h"
+
+namespace OpenLoco
+{
+#pragma pack(push, 1)
+    struct GameState
```

Comment:
This is a duplicate of the S5 struct, correct? I assume this is so we can deviate in the future?

Reply:
Yes that is correct.

Reply:
For the limits, you're tapping into the S5 namespace. Perhaps we can do the same thing here, instead of keeping an exact copy?